apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInAnyOrderCreatorSamples.kt

```diff
@@ -6,6 +6,28 @@ import kotlin.test.Test
 
 class MapLikeToContainInAnyOrderCreatorSamples {
 
+    @Test
+    fun entry() {
+        expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "b")
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "c")
```

Comment:
```suggestion
            expect(mapOf(1 to "a", 2 to "b"))
                .toContain.inAnyOrder.entry(2 to "c") // fails because the value "b" of key 2 (which exists in the subject) is not equal to "c"
```